examples/create-react-app-with-jss/package.json

```diff
@@ -3,13 +3,10 @@
   "version": "1.0.0",
   "private": true,
   "dependencies": {
-    "jss": "latest",
```

Comment:
No https://github.com/cherniavskii/material-ui/blob/da89e07aa14cac94ef268ab80c2dd98f2f60da2e/examples/create-react-app-with-jss/src/withRoot.js#L2

Reply:
But `jss` is listed in `material-ui` dependencies, which means that it is installed along with `material-ui` and can be imported from `node_modules` in any place within application

Reply:
It's a transitive dependency, it's brittle.